stdlib/printf.mli

```diff
@@ -102,7 +103,7 @@ val fprintf : out_channel -> ('a, out_channel, unit) format -> 'a
      space if positive.
    - [#]: request an alternate formatting style for the integer types
      ([x], [X], [o], [lx], [lX], [lo], [Lx], [LX], [Lo], [d], [i], [u],
-     [ld], [li], [lu], [Ld], [Li], [Lu], [nd], [ni], [nu]).
+     [ld], [li], [lu], [Ld], [Li], [Lu], [nd], [ni], [nu]) or float ([F]).
```

Comment:
Instead of `or float ([F])`, I'd rather be more explicit: `and the floating-point type [F]`.

Although this predates this change, I don't understand why the enumeration `[x], ... [nu]` is between parentheses.

Reply:
> Instead of or float ([F]), I'd rather be more explicit: and the floating-point type [F].

Right, done

> Although this predates this change, I don't understand why the enumeration [x], ... [nu] is between parentheses.

I guess because it is an enumeration of *all* integer formats (`nx`, `nX` and `no` are missing but actually supported). Since this complete enumeration is quite useless, I rather offer to remove it and add a slightly more precise documentation on the bahavior of `#` on each integer format.